### PR TITLE
Better CPU usage 2

### PR DESCRIPTION
**Pull request by KeitIG**
I removed the unwanted commits 

---

Still not perfect, but we're getting there.

Before: 
screen shot 2018-02-27 at 15 49 19

After:

screen shot 2018-02-27 at 15 34 26

For the record, when removing completely the animation: 

screen shot 2018-02-27 at 15 35 07

### CODE REVIEW

**Comment by KeitIG** [comment]
The plural is used because "informations" is what is meant.

**Comment by KeitIG** [comment]
Still not perfect, but we're getting there.

Before: 
screen shot 2018-02-27 at 15 49 19

After:

screen shot 2018-02-27 at 15 34 26

For the record, when removing completely the animation: 

screen shot 2018-02-27 at 15 35 07

**Comment by KeitIG** [comment]
I am still really uncertain about the last commit 
636b9ec

File: src/ui/components/Header/PlayingBarInfos.react.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      duration: null,
+      x: null,
+    });
+  }
+
+  render() {
+    const { trackPlaying } = this.props;
+    let elapsedPercent;
+
+    if (this.state.elapsed < trackPlaying.duration) elapsedPercent = this.state.elapsed * 100 / trackPlaying.duration;
```
Comment (YurySolovyov) [inline src/ui/components/Header/PlayingBarInfos.react.js]: Why do we only compute that conditionally?

Comment (KeitIG) [reply src/ui/components/Header/PlayingBarInfos.react.js]: This is a really good question, this looks weird

File: src/ui/components/Header/PlayingBarInfos.react.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    const bar = document.querySelector('.now-playing-bar');
+    const percent = ((e.pageX - (bar.offsetLeft + bar.offsetParent.offsetLeft)) / bar.offsetWidth) * 100;
+
+    const jumpTo = (percent * trackPlaying.duration) / 100;
+
+    AppActions.player.jumpTo(jumpTo);
+  }
+
+  dragOver(e) {
+    // Chack if it's needed to update currentTime
```
Comment (YurySolovyov) [inline src/ui/components/Header/PlayingBarInfos.react.js]: spelling: Ch**e**ck

Comment (KeitIG) [reply src/ui/components/Header/PlayingBarInfos.react.js]: u monster

**Comment by YurySolovyov** [comment]
> I am still really uncertain about the last commit 636b9ec

Me too, one way I can suggest to test this is to try to minimize player and see if it renders progress bar correctly after maximizing again.

**Pull request merged**